### PR TITLE
Links below title are now shown properly in mobile view

### LINKED ISSUE #85: Links below title are not proper in mobile view
screenshot from 2017-02-01 18-36-47

### PR DESCRIPTION
**Pull request by harshant**
changed the margin-right form 5px to -30 px

Closes #85

### CODE REVIEW

**Comment by pradeepgangwar** [comment]
@harshant Please refer to [this](http://api.coala.io/en/latest/Developers/Writing_Good_Commits.html) guide for writing good commit messages. Also you can enable gh-pages from your fork and paste the link here for other to view the changes you have made. That would make the review process smooth. Please do this.

**Comment by jayvdb** [comment]
Please fix your commit message

**Comment by harshant** [comment]
Added a new commit with some more effective changes and good updated commit message.

**Comment by pradeepgangwar** [comment]
@harshant Please squash your commits and use short and effective message inside body of commit message. Preferably use less than 60 words.

**Comment by harshant** [comment]
done

**Comment by pradeepgangwar** [comment]
@harshant Nice! squash also and it will be ready to land. Also please enable ghpages from your fork and paste the link here to test. It eases review process.

**Comment by harshant** [comment]
https://harshant.github.io/landing-frontend/#/home
Which commit you want to squash more ,I have squashed my second commit message.

**Comment by pradeepgangwar** [comment]
@harshant  Squashing does not mean shortening commit message. It is about wrapping more than 1 commits into a single commit. There should be only one commit per issue. Please read about squashing [here](https://git-scm.com/book/id/v2/Git-Tools-Rewriting-History)

**Comment by harshant** [comment]
Hey now everything looks fine ,squashed into one commit ,and here is the link
https://harshant.github.io/landing-frontend/#/home

**Comment by harshant** [comment]
Hey i want to work on another issue or review an issue , 
corobo assign https://github.com/coala/coala-quickstart/issues/190 says you are ineligible

**Comment by sims1253** [comment]
you did not reference any issue in the commit message but instead referenced this PR. 
Please reference the issue this belongs to.

**Comment by harshant** [comment]
looks like issue 238 is removed from the issue ,i am unable to find it

**Comment by harshant** [comment]
done

**Comment by RaiVaibhav** [comment]
Hello @harshant, just need a little rebase here and rest all the work is :+1:

**Comment by harshant** [comment]
hi, i am not very femiliar with rebase , could you please elaborate further what you want me to do?

**Comment by RaiVaibhav** [comment]
@harshant  please next time open a PR with different branch,
What I do is:
1.) Update my `master` branch
2.) Then update my `local` branch through `master` branch.

For [Git Basics please got through, please read all](http://api.coala.io/en/latest/Developers/Git_Basics.html)

Or [got through](https://stackoverflow.com/questions/7244321/how-do-i-update-a-github-forked-repository)
It will only help you in updating a `master` branch (Note: The changes you are making will reflect locally, to reflect on GitHub repo you have to `push your changes`.)

After that, if you have opened a PR through some other branch you have to update your local branch from master branch (I will recommend to not do any changes in your master branch).

Hoping it will help :)

**Comment by jayvdb** [comment]
@gitmate-ff rebase

**Comment by jayvdb** [comment]
Hey! I'm [GitMate.io](https://gitmate.io)! This pull request is being rebased automatically. Please **DO NOT** push while rebase is in progress or your changes would be lost permanently :warning:

**Comment by jayvdb** [comment]
Automated rebase with [GitMate.io](https://gitmate.io) was successful! :tada:

File: vendors/materialize/css/materialize.css [hunk 1]
```diff
@@ -5581,7 +5581,8 @@ button.btn-floating {
   border-radius: 16px;
   background-color: #e4e4e4;
   margin-bottom: 5px;
-  margin-right: 5px;
+  margin-right: -40px;
```
Comment (riouwa) [inline vendors/materialize/css/materialize.css]: Please never modify a vendor supplied file.

Find another method to edit our own style.

Possibly, you might want to see coalaCSS repo for this kind of issue.

**Comment by jayvdb** [comment]
Closing due to inactivity.

**Pull request closed without merging**